app/assets/stylesheets/govuk_publishing_components/components/_hint.scss

```diff
@@ -1 +1,5 @@
 @import "govuk/components/hint/hint";
+
+.govuk-hint {
+  display: block;
```

Comment:
Is this needed? `.govuk-hint` is a div element, which is block by default.

Reply:
That's interesting, actually when the `.govuk-hint` class is used in the label component (though not in the other classes) it is a span. If it is an inline element the text floats to the left and looks wrong (which was the reason for adding it). I've changed it to only apply where it is used in the label component.

Reply:
Yes, I agree, that would be the better solution. I've updated that to a div and removed the addition to the CSS.